.circleci/config.yml

```diff
@@ -78,6 +78,7 @@ jobs:
            EOF
 
            sudo service docker restart
+     - run: dig go.googlesource.com A go.googlesource.com AAAA # Debug IPv6 network issues
```

Comment:
Though, I wonder if the problem is actually within docker, and not outside... Maybe I should put this in both places?

Reply:
More places the better insight..